engines/gob/hotspots.cpp

```diff
@@ -728,6 +737,15 @@ uint16 Hotspots::check(uint8 handleMouse, int16 delay, uint16 &id, uint16 &index
 		_currentIndex = 0;
 	}
 
+	if (_vm->getGameType() == kGameTypeAdibou2 &&
+		_currentKey != 0 &&
```

Comment:
```suggestion
			_currentKey != 0 &&
```
And below, so there is a visual distinction between condition continuation and the condition body.